packages/utils-debugging-protocol-common/src/debugging-protocol-connector.ts

```diff
@@ -136,7 +127,7 @@ export class Connector implements IConnector {
              * doesn't need to be escaped.
              */
             const query: string = `[src$="${basename}" i],[href$="${basename}" i],[src$="${decodeURIComponent(basename)}" i],[href$="${decodeURIComponent(basename)}" i]`;
-            const newElements: Array<AsyncHTMLElement> = await this._dom.querySelectorAll(query);
+            const newElements: Array<AsyncHTMLElement> = await this._dom!.querySelectorAll(query);
```

Comment:
Nit: `this._dom!` is technically safe as all callers (or callers of callers) ensured it existed before we got here. However, we're so far removed by this point it's hard to tell without reading a fair amount of code. 

What do you think about adding `dom: CDPAsyncHTMLDocument` as a parameter to this function and having it explicitly passed down from the callers that performed the check to ensure it existed?